### ISSUE TITLE
Word problems !

### ISSUE THREAD

**Issue by tmac21** [jangxx/netflix-1080p#26]
Hello sir, I have some problems. When I watch Netflix all videos has these problems like this pic. If you check the pics you can see.
What can you do about it? I mean, the initial is occasionally separated from the word and the punctuation that follows is also at the bottom.

https://www.hizliresim.com/dc7hwpp

https://www.hizliresim.com/6zev8sg

**Comment by jangxx** [comment]
I don't think that your problem originates from this extension. Have you tried to change the size of the subtitles in the settings?
